### PR TITLE
[candi] add step for updating legacy tfadm

### PR DESCRIPTION
**Pull request by sprait**
## Description

Add step for updating legacy tfadm key.

## Why do we need it, and what problem does it solve?

Update tfadm key.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: candi
type: fix
summary: update tfadm pub key.
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/openstack'...

**Comment by z9r5** [comment]
/e2e/abort openstack;Standard;containerd;1.25 8157566938 8157566938-1-con-1-25 /sys/deckhouse-oss/install:pr7714 debian@31.129.32.28

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
some comments

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/aws'...

File: ee/fe/candi/bashible/common-steps/all/099_upd_tfadm.sh.tpl [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+TFADM_PUB='cert-authority,principals="tfadm" ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDIg88Qs3x5KR6IQnKYgif1hCig08hYh4eHrriCp4916KpjocdYgq7/TWTeeiJktEEkSo0E+8VJjW0IRk5qdBzPPGRWeSpktf18u4lrq5gWC1OkdDnIgioncxGihA1+ueclQj+eaJZRSh5E8//AROKBJSfAeN6sbn0SqCbZj3cqjR8bgDe4UgdPyXOpJ01iVWgzB0Bk8FHX3NRHdVwZDt7pBRX5Dwvd8z5kifxfWVOC0M5l9tH4dvdHNgthVYTTrIxC7fpAMNul4foe1+2iMg0GAe25MMmlgquWOK91yT2VSfvngsXpx8vb0juKS3Peq/B3TjAjmLxJBVxXx/rQA+AF7J1m6WgCB1bGNPl6Yc86v3SUdScACX+Qj147E5EzsCZbXRDiTq3GQQ9EnnmkK73obZrHkCeSSbH2rWln1IzzbgCc1R0yXpBT1Scjyx0ld31C5cSNq+sL21C0govCtSa93hgxggBUFdQ2a4c7U9knNB4khOF7nTtK543NDsJhXByjF6D484LTSV6Vrg5dbGFFxolaS++rgThtyUkH8BecuKFtxqwx+qxYlqAqO0E1QaUfEXbYtDQAFbcn6AcgU+M633uxzmvp9IYeAp6cy7T5Bbwu+jsErkviEkOwrASTMkJP4IXT6A3RLMNO0TvQwd9vP+kYL8Nk7a2QEHCCfwvpFw=='
+
+auth_keys=$(grep -lirs "${TFADM_OLD_PUB}" {/root,/home/*,/home/*/*}/.ssh/authorized_keys || true )
```
[the remark this answers is not in the document]
Comment (sprait) [reply ee/fe/candi/bashible/common-steps/all/099_upd_tfadm.sh.tpl]: We can use /var/lib/extrausers/passwd, so i tried to cover this case. 
But I found other decision with `getent passwd` which get summary passwd files, pls, recheck it.

**Review by name212 — APPROVED** [review]

**Review by RomanenkoDenys — APPROVED** [review]

**Pull request merged**